opennms-webapp/src/main/webapp/about/index.jsp

```diff
@@ -131,10 +131,25 @@
     <th>Database Version:</th>
     <td><%=dbVersion%></td>
   </tr>
+<%
+    String globalMBStrategy, sinkMBStrategy, rpcMBStrategy;
+    globalMBStrategy = org.opennms.core.rpc.common.RpcStrategy.getRpcStrategy().getDescr();
```

Comment:
Think you need to import these packages.

Reply:
Importing doesn't seem to do the job; it still breaks on build and I get an additional message:

```
An error occurred at line: [21] in the generated java file: [/Users/jeffg/git/foundation-work/opennms-webapp/target/jsp-classes/org/apache/jsp/about/index_jsp.java]
Only a type can be imported. org.opennms.core.ipc.sink.common.SinkStrategy resolves to a package
```

Same result with either foundation-2019 or foundation-2020. Does the `opennms-webapp` POM perhaps need updating?

Reply:
Guess you have to bring both dependencies into `opennms-webapp` and import  the packages [here](https://github.com/OpenNMS/opennms/blob/b0a7c12a3c3bf81f5886b1388b5488c918235e31/opennms-webapp/src/main/webapp/about/index.jsp#L35)

Reply:
Got both dependencies added and ran it through full smoke, which passed. I think this should do the job.
Also changed from `descr` to `name` in the page.